### ISSUE TITLE
annotation files (poly version)

### ISSUE THREAD

**Issue by pieterblok** [waiyulam/ORCNN#1]
@waiyulam 

Thank you for this repository. I was able to run your code, however I miss two files that I cannot find online:
datasets/coco/annotations/COCO_amodal_train2014_with_classes_poly.json
datasets/coco/annotations/COCO_amodal_val2014_with_classes_poly.json

I have two similar files from https://github.com/Wakeupbuddy/amodalAPI, however they miss the right categories. 

Can you help me?

**Comment by trqminh** [comment]
Hi @pieterblok,
Have you gotten the right annotation files yet? If yes, could you share them with me or tell me how to download them?
Thank you for your consideration

**Comment by pieterblok** [comment]
> Hi @pieterblok,
> 
> Have you gotten the right annotation files yet? If yes, could you share them with me or tell me how to download them?
> 
> Thank you for your consideration

@trqminh No, unfortunately I didn't get the annotation files. I was in email contact with waiyulam  but he couldn't share the files.

If you are interested to get ORCNN working with your own dataset, please have a look at my git repository: https://git.wur.nl/blok012/sizecnn. This repository also includes an instruction on how to annotate the images with LabelMe and how to convert the annotations to COCO-ORCNN format.

Good luck!
